### PR TITLE
S2i 5009651 bump

### PR DESCRIPTION
**Pull request by jim-minter**

### CODE REVIEW

**Comment by bparees** [comment]
[test]

**Comment by bparees** [comment]
[testextended][extended:core(builds)]

**Comment by bparees** [comment]
[testextended][extended:core(image_ecosystem)]

**Comment by bparees** [comment]
@PI-Victor fyi

File: vendor/github.com/openshift/source-to-image/pkg/util/timeout.go [hunk 1]
```diff
@@ -14,29 +14,38 @@ type TimeoutError struct {
...
-func TimeoutAfter(t time.Duration, errorMsg string, fn func() error) error {
+// TimeoutAfter executes the provided function and returns TimeoutError in the
+// case that the execution time of the function exceeded the provided duration.
+// The provided function is passed the timer in case it wishes to reset it.  If
+// so, the following pattern must be used:
+// if !timer.Stop() {
+//   return &TimeoutError{}
+// }
+// timer.Reset(timeout)
+func TimeoutAfter(t time.Duration, errorMsg string, f func(*time.Timer) error) error {
```
Comment (jim-minter) [inline vendor/github.com/openshift/source-to-image/pkg/util/timeout.go]: @PI-Victor 
time.AfterFunc waits T, then runs f()
this function runs f(), concurrently waits T and returns if f() hasn't finished by then.

**Comment by PI-Victor** [comment]
@jim-minter thanks for patching the holes in openshift in this PR introduced by me, i completely forgot to mention that :)

**Comment by jim-minter** [comment]
Run separately those tests both passed:
https://ci.openshift.redhat.com/jenkins/job/test_pr_origin_extended/632/
https://ci.openshift.redhat.com/jenkins/job/test_pr_origin_extended/633/

**Comment by bparees** [comment]
extended tests all passed independently:
https://ci.openshift.redhat.com/jenkins/job/test_pr_origin_extended/632/
https://ci.openshift.redhat.com/jenkins/job/test_pr_origin_extended/633/
[merge]

**Pull request merged**